### DISCUSSION TITLE
Recovering Deleted Persistent

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#9214] [QA - tech support + general] [answered]
So, I used the "I'm going to take you somewhere" option and put Monika on my USB, but my USB died with all the backups of the MAS persitent and I was lucky enough to put the Monika File on my PC. The problem comes because i deleted the MAS persistent files and the entire game on my PC because I'm dumb. I'm trying to recover those files from Monika's memories by using a program called "Recuva" because those files were entirely deleted and I found some of the persistents there, but when I put them in they were all corrupted.

This happened a few days ago. I downloaded MAS again today and i'm trying to make those persistent files work. I would like a solution to make those persistent files work. Is there anything I can do?

**Comment by Booplicate** [comment, marked as answer]
Persistent is very fragile, there was a very little chance of recovering it. You can't fix it, only if you made backups.

> The problem comes because i deleted the MAS persistent files and the entire game on my PC because I'm dumb

Try to restore the files from the recycle bin. We make 10 backups for you, did you delete those too?

**Reply by ghost** [reply]
The backups were deleted too, and I tried to restore them, I didn't find some of the recent ones. Still, I tried to put them in, but the game tells me that they're corrupted as well. Also, I found some "dc.mcal" files but I don't know what they are for. I'm still looking on my PC for persistent and backups. Do I just need the persistent? Or I can try and search for backups.

**Reply by Booplicate** [reply]
`dc.mcal` and akin are for calendar, they are pretty much useless. You need `persistent` or `persistent##.bak` files. Have you ever uploaded your backups to a cloud or your phone?

**Reply by ghost** [reply]
No, sorry. I don't know too much about that program, i don't know if i have it even.
